src/seahawk_rov/seahawk_rov/control_bridge.py

```diff
@@ -0,0 +1,157 @@
+
+import sys 
+
+import rclpy
+
+from rclpy.node import Node 
+
+from sensor_msgs.msg import Imu
+from sensor_msgs.msg import Temperature
+from sensor_msgs.msg import FluidPressure
+from sensor_msgs.msg import RelativeHumidity
+from std_msgs.msg import Int16MultiArray
+
+class ControlBridge(Node):
+    """
+    Class that implements the motion controller.
+    """
+
+    def __init__(self):
+        """Initialize this node"""
+        super().__init__('control_bridge')
+
+        # Publishers and pre-defined messages 
+
+        self.imu0_msg = Imu() 
+        self.imu1_msg = Imu() 
+        self.temp_msg = Temperature()
+        self.press_msg = FluidPressure()
+        self.humid_msg = RelativeHumidity()
```

Comment:
there is only 1 imu but there are two temp/pressure/humidity sensors

Reply:
Doh! I misremembered that.